dom/schema/v1/tollgate_crossing.sql

```diff
@@ -0,0 +1,9 @@
+create table if not exists tollgate_crossing
+(
+    id          text not null primary key,
+    tollgate_id text,
+    driver_id   text,
+    crossing    jsonb,
+    created_at  timestamptz,
+    constraint fk_tollgate foreign key (tollgate_id) references tollgate (id) on delete CASCADE
```

Comment:
This gives an error, since `tallgage` table does not exists

Reply:
the order of applying schema files: 
- tollgate.sql
- tollgate_crossing.sql